### PR TITLE
Add units

### PR DESCRIPTION
**Pull request by mrvdb**
Just added powerpc64le arch

### CODE REVIEW

**Comment by mrvdb** [comment]
What is the etiquette here? Do I add myself as Maintainer line? How do we distinguish maintainers?

**Comment by FabioLolix** [comment]
Like this seems https://github.com/kth5/archpower/blob/master/a52dec/PKGBUILD

```
# POWER Maintainer: 
# Maintainer: 
# Maintainer: 
# Contributor: 
```

File: units/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Maintainer: Kyle Keen <keenerd@gmail.com>
```
[the remark this answers is not in the document]
Comment (mrvdb) [reply units/PKGBUILD:2]: Added the requested changes.

**Comment by kth5** [comment]
Timed out I guess

**Pull request closed without merging**